core/src/androidMain/kotlin/com/revenuecat/purchases/kmp/models/PricingPhase.android.kt

```diff
@@ -1,5 +1,22 @@
+@file:Suppress("EXTENSION_SHADOWED_BY_MEMBER")
+
 package com.revenuecat.purchases.kmp.models
 
-import com.revenuecat.purchases.models.PricingPhase as RcPricingPhase
+import com.revenuecat.purchases.models.PricingPhase as AndroidPricingPhase
+
+public actual typealias PricingPhase = AndroidPricingPhase
+
+public actual val PricingPhase.billingPeriod: Period
+    get() = billingPeriod
+
+public actual val PricingPhase.recurrenceMode: RecurrenceMode
+    get() = recurrenceMode.toRecurrenceMode()
```

Comment:
I had to move `PricingPhase` to extensions in order to do this.

Reply:
Nevermind, I'll just revert it for now. Less decision-making for everyone.

Reply:
PR with the revert opened: https://github.com/RevenueCat/purchases-kmp/pull/141